Makefile

```diff
@@ -423,10 +423,12 @@ utils/config.ml: utils/config.mlp config/Makefile
 	    -e 's|%%ARCH%%|$(ARCH)|' \
 	    -e 's|%%MODEL%%|$(MODEL)|' \
 	    -e 's|%%SYSTEM%%|$(SYSTEM)|' \
+	    -e 's|%%EXT_EXE%%||' \
```

Comment:
That's a good point, I'll change it to be whatever configure decided it would use for the installed binaries.